packages/core/src/extensions/lens-extension.ts

```diff
@@ -80,6 +87,11 @@ export class LensExtension<
     return this.manifest.description;
   }
 
+  // Name of extension for persisting data
+  get storeName() {
+    return this.manifest.storeName || this.name;
```

Comment:
_nit:_ 
```suggestion
    return this.manifest.storeName ?? this.name;
```

Reply:
I don't think we want this. If `storeName` is set as `""`, we want to fall back to `name`.